scripts/qwik-city.ts

```diff
@@ -402,11 +407,11 @@ async function buildAdapterCloudRunVite(config: BuildConfig, inputDir: string, o
 }
 
 async function buildAdapterExpressVite(config: BuildConfig, inputDir: string, outputDir: string) {
```

Comment:
shouldn't we add a new `buildAdapterNodeServer` instead o replacing express? otherwise we will not generate the files, that the package.json is pointing to